src/main/app.ts

```diff
@@ -35,20 +38,24 @@ export async function initApp() {
     app.disableHardwareAcceleration()
   }
 
-  app.on('second-instance', () => {
+  app.on('second-instance', (_, argv) => {
     const mainWindow = getMainWindow()
-
     if (mainWindow) {
       if (mainWindow.isMinimized()) {
         mainWindow.restore()
       }
 
       mainWindow.show()
+
+      const mailto = argv[argv.length - 1]
+      if (mailto.startsWith('mailto:')) {
+        handleMailto(mailto)
+      }
```

Comment:
Since `argv` is only handled when trying to launch a second instance, what if the launch is the first instance?

Reply:
Good catch, addressed :wink: